### ISSUE TITLE
залоги после основ на к, х, ғ, г, ң

### ISSUE THREAD

**Issue by veramal** [tuffnatty/suddenly#213]
Есть сейчас такой разбор:
турлаанзар
v турлағарға ‛остепениться’ турлах
-(Ы)н-------СА----------(Ы)ңАр---
-Refl-------Cond----------2pl.br---

Хорошо ли, что основа на -_х_ стягивается с рефлексивом? Судя по тому, что на -_аанарға_ в словаре не кончается ни одна лексема, вряд ли это допустимый вариант.

**Comment by adybo** [comment]
Можем сейчас убить, а потом какой-нибудь кызыльский вылезет.

**Comment by veramal** [comment]
Нет, давайте все-таки убьем. Например, слово оолах 'мальчик' имеет два паразитических разбора с пассивом:
v оорға ‛линять’ оғ
-(Ы)л-----------------АК---
-Pass-----------------ImpInclDial---

v оңарға ‛выцветать’ оң
-(Ы)л-----------------АК---
-Pass-----------------ImpInclDial---

Раз пока не встретилось случаев стяжения с основами на к, г, ң, то и дальше, думаю, не встретится.

**Comment by veramal** [comment]
Дописала в правило 3.4. на с. 53, что при выпадении _к, х_ имеются в виду только именные основы и аффиксы посессива (это поможет словам типа _турлах_). В статье 2019 г. упомянуто, что других случаев нам не встретилось.
Двубуквенные глаголы на _-Г, -ң_ точно не могут сокращаться. С трехсложными стяжение, кажется, обязательно - ср. _чуунарға/?чуғынарға_ 'мыться' (< _чуғ_ 'мыть'). В словаре все случаи основ на _-ғын_, _-ғыл_, _-ғыс_ произведены от корней с конечным -К. Основы со стяжением, наоборот, все произведены от корней с конечным -Г, например: _сааларға_ 'доиться', _саазарға_ 'помогать доить' (< _сағ_ 'доить'), _моонарға_ / _поонарға_ 'вешаться' (< _поғ_ 'душить'), _нууларға_ 'закрываться' (о глазах) (< _нуғ_ 'жмурить'). Дописала про стяжение с залогами в правило 3.1 на с.53.

**Comment by adybo** [comment]
Да, вроде у нас и было это для односложных корней?

  
>Воскресенье, 26 января 2025, 19:21 +03:00 от veramal ***@***.***>:
> 
> 
>Дописала в правило 3.4. на с. 53, что при выпадении  к, х имеются в виду только именные основы и аффиксы посессива (это поможет словам типа  турлах ). В статье 2019 г. упомянуто, что других случаев нам не встретилось.
>Двубуквенные глаголы на  -Г, -ң точно не могут сокращаться. С трехсложными стяжение, кажется, обязательно - ср.  чуунарға/?чуғынарға 'мыться' (<  чуғ 'мыть'). В словаре все случаи основ на  -ғын ,  -ғыл ,  -ғыс произведены от корней с конечным -К. Основы со стяжением, наоборот, все произведены от корней с конечным -Г, например:  сааларға 'доиться',  саазарға 'помогать доить' (<  сағ 'доить'),  моонарға /  поонарға 'вешаться' (<  поғ 'душить'),  нууларға 'закрываться' (о глазах) (<  нуғ 'жмурить'). Дописала про стяжение с залогами в правило 3.1 на с.53.
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>You are receiving this because you commented. Message ID:  <tuffnatty/suddenly/issues/213/2614491743 @ github . com> 
 
 
--
Anna Dybo
Отправлено из Почты  Mail

**Comment by veramal** [comment]
а) надо различать односложные на -к и односложные на -г, б) надо различать VC и CVC. Стяжение возможно (и обязательно) только для CVC на -г, прочим нужно запретить.

**Comment by veramal** [comment]
турлаанзар и оолах не воспроизводят разборы с глагольной основой, остальные упомянутые глаголы разбираются с выделением залоговых аффиксов.
